### PR TITLE
Doc/update install and usage

### LINKED ISSUE #139: Edit the README to better explain how to run the program
My team experimented with running the program to test features. As a result, we felt that the instructions in the README were confusing at some points. To fix this we will adjust thee instructions to increase the clarity and allow other users to simply run the program.

### LINKED ISSUE #162: Update Installation and and Initial Setup Sections of README
Currently the installation and initial setup sections of the README are out of date and refer to `pip`, `gspread`, etc, which are no longer used by GatorGrouper. We should update the README to accurately reflect the current installation and initial setup steps.

### PR DESCRIPTION
**Pull request by paladp**
# Description of Pull Request

This pull request updates the README for the GatorGrouper project to reflect the changes made to the installation process. The initial descriptions were taken from the installation instructions provided by [GatorGrader](https://github.com/GatorEducator/gatorgrader) and modified to fit GatorGrouper. The commands provided in the README were given under the assumption that the user was in the root directory of the project. Additionally the testing section was removed and replaced with a shorter section discussing `pytest`. Tagging some of the testing team so they can make the changes necessary to the testing section.

Fixes #139 
Fixes #162

## Type of Change

Please describe the pull request as one of the following: 

- [ ] Bug fix
- [ ] Breaking change
- [ ] New feature
- [x] Documentation update
- [ ] Other 

## Tags
 
@Lancasterwu @ilikerustoo @Michionlion @aubreypc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,33 +62,50 @@ After you finish this process, you should be able
...
 `Python 3.6.8` in your terminal.
 
-## Initial Setup
+As a Python 3 program, GatorGrouper relies on
+[Pipenv](https://github.com/pypa/pipenv) for the installation of the libraries
+on which it depends and the creation of the virtual environments in which it
+runs. To install GatorGrader, you should first follow Pipenv's installation
+instructions. You should also ensure that you have installed Git on your
+computer and that you can run Git commands in a terminal window. Then, you can
+type the following command in your terminal window to clone GatorGrader's GitHub
```
Comment (Michionlion) [inline README.md]: `GatorGrader` is mentioned multiple times in this area.

File: README.md [hunk 2]
```diff
@@ -90,33 +62,50 @@ After you finish this process, you should be able
...
+successfully, that's all you you have to do to install GatorGrouper!
 
-```shell
-python3 -m pip install --user gspread oauth2client
+## Testing GatorGrouper
+
+GatorGrader uses [Pytest](https://docs.pytest.org/en/latest/) for testing.
+Depending on your goals, there are several different configurations in which you
+can run the provided test suite. If you want to run the test suite to see if all
+of the test cases are correctly passing, then you can type one of the following
```
Comment (Michionlion) [inline README.md]: This says "one of the following" but there is only one example following.

File: README.md [hunk 3]
```diff
@@ -128,22 +117,32 @@ file called `students.csv` and creating groups of two students means that you
...
+```
+
+### Specify File Containing List of Students
+
+If your `.csv` file is not named `students.csv`, use the `--students-file` flag
+to let GatorGrouper know the file you are using.
+
+```shell
+pipenv run python3 gatorgrouper/utils/gatorgrouper.py --students-file "file_name.csv"
+pipenv run python3 gatorgrouper/utils/gatorgrouper.py --students-file file_name.csv
```
Comment (Michionlion) [inline README.md]: This command is duplicated.

File: README.md (hunk 1 above)
Comment (huangs1) [reply README.md]: This should be resolved.

File: README.md (hunk 2 above)
Comment (huangs1) [reply README.md]: This should be resolved.

File: README.md (hunk 3 above)
Comment (huangs1) [reply README.md]: This should be resolved.

**Comment by huangs1** [comment]
@Michionlion This PR should be ready to merge.

File: README.md [hunk 4]
```diff
@@ -63,9 +63,9 @@ to type `python --version` and get `Python 3.7.2` or other version after
 `Python 3.6.8` in your terminal.
 
 As a Python 3 program, GatorGrouper relies on
-[Pipenv](https://github.com/pypa/pipenv) for the installation of the libraries
-on which it depends and the creation of the virtual environments in which it
-runs. To install GatorGrouper, you should first follow Pipenv's installation
+[Pipenv](https://github.com/pypa/pipenv) for the installation of the libraries that
+it depends on and the creation of the virtual environments in which it
+runs in. To install GatorGrouper, you should first follow Pipenv's installation
```
Comment (Michionlion) [inline README.md]: This addition of `in` is redundant.

File: README.md [hunk 5]
```diff
@@ -76,11 +76,11 @@ git clone git@github.com:GatorEducator/gatorgrouper.git
 ```
 
 If you plan to develop new features for GatorGrouper or if you want to run the
-tool's test suite in [Pytest](https://github.com/pytest-dev/pytest), then you
-will need to install the developer dependencies by typing `pipenv install --dev`
+tool's test suite in [Pytest](https://github.com/pytest-dev/pytest), then you should
+install the developer dependencies by entering `pipenv install --dev`
```
Comment (Michionlion) [inline README.md]: This should probably be `running` or reworded to use `run`.

**Pull request merged**